package.json

```diff
@@ -84,9 +85,13 @@
     "lint:fix": "yarn run lint --fix"
   },
   "lint-staged": {
-    "{packages,examples}/**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
+    "{packages,examples}/**/*.{json,css,scss,md}": [
       "prettier --write",
       "git add"
+    ],
+    "{packages,examples}/**/*.js,jsx,ts,tsx": [
+      "eslint --fix",
```

Comment:
What happens if eslint cannot fix these? Usually prettier only includes stylistic changes whereas eslint can make other changes. Also we miss out on getting the prettier changes.

Do we want to look into https://github.com/prettier/prettier-eslint instead?